.env.example

```
@@ -85,6 +86,8 @@ ANTHROPIC_BASE_URL=https://ai.angrist.net
 CLAUDE_SEARCH_MODEL=claude-haiku-4-5-20251001
 OPENAI_BASE_URL=https://ai.angrist.net/v1
 CODEX_SEARCH_MODEL=gpt-5.6-luna
+Z_AI_BASE_URL=https://api.z.ai/api/coding/paas/v4
+ZAI_SEARCH_MODEL=glm-4.6
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ZAI_SEARCH_MODEL key should go before the Z_AI_BASE_URL key

(UnorderedKey)